### PR TITLE
fix: setOptions() should use forceUpdate

### PR DESCRIPTION
**Pull request by atomiks**
Makes `createPopper` sync

### CODE REVIEW

File: src/types.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ export type Instance = {|
   destroy: () => void,
   forceUpdate: () => void,
   update: () => Promise<void>,
-  setOptions: (options: $Shape<Options>) => void,
+  setOptions: (options: $Shape<Options>) => Promise<void>,
```
Comment (FezVrasta) [inline src/types.js]: I think this should be Promise<State>

Comment (atomiks) [reply src/types.js]: Should `update` have that as well?

Comment (FezVrasta) [reply src/types.js]: yes

Comment (FezVrasta) [reply src/types.js]: While you are there, could you also update the docs types?

**Comment by FezVrasta** [comment]
Manually merged, thanks

**Pull request closed without merging**